src/components/ObservationsFlashList/ObservationsFlashList.js

```diff
@@ -251,7 +251,8 @@ const ObservationsFlashList: Function = forwardRef( ( {
 
   const extraData = {
     gridItemWidth,
-    numColumns
+    numColumns,
+    layout
```

Comment:
awwwwwwyeah, I love a good #OLC (one line commit) fix ; D

LGTM but it's maybe worth testing with Johannes's flash list update? https://github.com/inaturalist/iNaturalistReactNative/pull/3163

Reply:
closed as completed! ; D